ebml/EbmlDummy.h

```diff
@@ -44,7 +44,6 @@ class EBML_DLL_API EbmlDummy : public EbmlBinary {
   public:
     EbmlDummy() :DummyId(DummyRawId) {}
     EbmlDummy(const EbmlId & aId) : DummyId(aId) {}
-    EbmlDummy(const EbmlDummy & ElementToClone) = default;
```

Comment:
The commit says add functions but most of it is removal of methods. It looks like 2 commits merged into one.

Reply:
Somewhat. The commit was made to satisfy https://en.cppreference.com/w/cpp/language/rule_of_three